rudder-web/src/main/webapp/bootstrap-3.2.0-dist/css/tw-bs-bootstrap.css

```diff
@@ -2441,7 +2441,7 @@
 }
 .tw-bs .form-control {
   display: block;
-  width: 100%;
+  /*width: 100%;*/
```

Comment:
You should not do modification on tw-bs-bootstrap.css file

Please do them in rudder.css or better rudder-bootstrap.css